lib/db/mysql.js

```diff
@@ -701,6 +701,35 @@ module.exports = function (log, error) {
     return this.write(UPDATE_LOCALE, [data.locale, uid])
   }
 
+  var CREATE_UNBLOCK_CODE = 'CALL createUnblockCode_0(?, ?, ?)'
+
+  MySql.prototype.createUnblockCode = function (uid, code) {
+    return this.write(
+      CREATE_UNBLOCK_CODE,
+      [ uid, code, Date.now() ],
```

Comment:
I wonder whether it would be worth storing a hash of the code, rather than the raw code, to guard against things like https://bugzilla.mozilla.org/show_bug.cgi?id=1305337 and to ensure that an accidental db leak doesn't give you the power to use any existing unblock codes.